calicoctl/commands/node/diags.go

```diff
@@ -26,6 +26,8 @@ import (
 	"github.com/docopt/docopt-go"
 	log "github.com/sirupsen/logrus"
 	shutil "github.com/termie/go-shutil"
+	"bufio"
```

Comment:
nit: not sure if you ran `goimports` on this, but the recommendation is to have all the Go native imports together at the top